backend/handler/igdb_handler.py

```diff
@@ -114,11 +116,20 @@ def get_platform(self, p_slug: str):
 
     @check_twitch_token
     def get_rom(self, file_name: str, p_igdb_id: int):
-        search_term = uc(get_search_term(file_name))
+        search_term = get_search_term(file_name)
+
+        # Patch support for PS2 OPL filename format
+        match = re.match(ps2_opl_regex, search_term)
+        if p_igdb_id == 8 and match:
+            serial_code = match.group(1)
+            index_entry = opl_index.get(serial_code, None)
+            if index_entry:
+                search_term = index_entry["Name"]
```

Comment:
The change is fairly self-contained, so as not to pollute the rest of the code base.